### ISSUE TITLE
Uncaught error in IE11: SCRIPT5045 Assignment to read-only properties is not allowed in strict mode`

### ISSUE THREAD

**Issue by LarsVonQualen** [webcomponents/custom-elements#184]
Hi,
so I have to support IE11, i know, not ideal.

When i include the webcomponents-bundle.js file, IE will throw this error
`IE11: SCRIPT5045: Assignment to read-only properties is not allowed in strict mode`
here:
[https://github.com/webcomponents/custom-elements/blob/master/src/CustomElementRegistry.js#L267](https://github.com/webcomponents/custom-elements/blob/master/src/CustomElementRegistry.js#L267)

I'm not sure exactly why, or if this is even an issue, everything seems to actually work, it's just an annoyance, and i guess there's the risk of IE showing an unexpected error popup to the user.

It seems to me that the error should be handled gracefully thus not ending in console output.

Any thoughts? Or maybe even a way to get the error to go away? 😄

**Comment by LarsVonQualen** [comment]
So this would be triggered if i have some JS that tries to assign anything to the style property of the custom element?

**Comment by web-padawan** [comment]
Yes, you can still use e.g.`element.style.display = 'none'` but not reassign the whole property.
I'm not sure if that's the case for you, it's just the only one I have encountered so far.

**Comment by azakus** [comment]
Can you provide an reproduction of the error?
